src/scene.ts

```diff
@@ -763,6 +763,54 @@ export class Scene extends AbstractScene implements IAnimatable, IClipPlanesHold
         InputManager.ExclusiveDoubleClickMode = value;
     }
 
+    /**
+     * Bind the current view position to an effect.
+     * @param effect The effect to be bound
+     * @param scene The scene the eyes position is used from
+     * @param variableName name of the shader variable that will hold the eye position
+     * @param isVector3 true to indicates that variableName is a Vector3 and not a Vector4
+     * @return the computed eye position
+     */
+     public static BindEyePosition(effect: Nullable<Effect>, scene: Scene, variableName = "vEyePosition", isVector3 = false): Vector4 {

```

Comment:
Why static if we pass the scene in ?

Reply:
I preferred to retain the exact same signature to avoid too many breaking changes, but the move is a breaking change itself so you're right it's better to just make it non static. Going to perform the change.